docker/tc_cert/generate-certificates.cmd

```diff
@@ -0,0 +1,1052 @@
+@echo off
```

Comment:
I will send you latest version of the script.
Also linux versions, plus script for renewing certificates.